### DISCUSSION TITLE
Python library for oqs-openssl?

### DISCUSSION THREAD

**Discussion by gobbledy-gook** [open-quantum-safe/oqs-provider#506] [Q&A] [answered]
I am working on a project which requires post-quantum modified OpenSSL. From what I have seen, oqs-provider is a CLI tool, and I want a Python library like the one we have for original OpenSSL using which I can make a PQ_TLS connection with the test server provided by Open-Quantum-Safe. So my question is

- Is such a Python library by liboqs for making a PQ_TLS connection available?

My background: I am a final-year student at an NIT in India. I might have missed something that led me to this question. Any help would be highly appreciated.

**Comment by baentsch** [comment, marked as answer]
Take a look at https://github.com/open-quantum-safe/liboqs-python/blob/main/docker%2FREADME.md. That basically builds what you are looking for.

**Reply by gobbledy-gook** [reply]
Okay that helps, Thanks @baentsch !
